packages/runtime/src/useCases/consumption/attributes/AttributeSuccessionResponse.ts

```diff
@@ -1,6 +1,13 @@
+import { CoreId } from "@nmshd/transport";
 import { LocalAttributeDTO } from "../../../types";
 
 export interface AttributeSuccessionResponse {
     predecessor: LocalAttributeDTO;
     successor: LocalAttributeDTO;
 }
+
+export interface AttributeSuccessionWithNotificationResponse {
```

Comment:
you export this type to use it only in the tests, why?

Reply:
I think you can just use `NotifyPeerAboutRepositoryAttributeSuccessionResponse` everywhere where you used `AttributeSuccessionWithNotificationResponse` and `AttributeSuccessionResponse` and then remove this entire file :)

Reply:
Right, initially (in cns-runtime) I used AttributeSuccessionWithNotificationResponse as Reponse type for the use cases NotifyPeerAboutRepositoryAttributeSuccession and SucceedRelationshipAttributeAndNotifyPeer.  I realized that I needed it for a helper, which handles RepositoryAttributes, as well as RelationshipAttributes. Hence, it didn't seem right to me to use one of both reponses, but instead implement a new one and use it in the use cases as well. Would it be agreeable to keep AttributeSuccessionWithNotificationReponse and use it in the two use cases?

AttributeSuccessionReponse was already there. It implements the same than SucceedRepositoryAttributeReponse. I guess this can be deleted and SucceedRepositoryAttributeReponse can be used instead.